src/components/Avatar.tsx

```diff
@@ -9,11 +10,26 @@ type AvatarProps = {
 
 // name = friendlyName ?? identity
 const Avatar: React.FC<AvatarProps> = ({ name, size }) => {
-	if (name.startsWith("whatsapp:") || name.startsWith("sms:") || name.startsWith("+")) {
-		return <PasteAvatar size={size ?? "sizeIcon70"} variant="user" name={name} icon={UserIcon} />
-	}
-	return <PasteAvatar size={size ?? "sizeIcon70"} variant="user" name={name} />
-	// use src for specified avatar images - once we have them!
+  if (
+    name.startsWith("whatsapp:") ||
+    name.startsWith("sms:") ||
+    name.startsWith("+")
+  ) {
+    return (
+      <PasteAvatar
+        size={size ?? "sizeIcon70"}
+        variant="user"
+        name={name}
+        icon={UserIcon}
+      />
+    );
+  }
+  return (
+    <Tooltip text={name} placement={"bottom-start"}>
```

Comment:
I don't know if we always want a tooltip here. Let me know what you think and I can make it conditional according to the props.

Reply:
We might want a tooltip for the sms participants as well, otherwise it's hard to distinguish them.

Reply:
Gotcha, will add.

Reply:
Done! Screenshot below